File: 04PHP_Curso_DAW/PHP/Condicionales(1-22)/Ejercicio14.php
<?php
/** 
 *  14.- Haz un programa que pida un  número de mes entre 1 y 12. A continuación devolverá el 
 *  nombre del mes correspondiente. 
 */
?>
<!DOCTYPE html>
<html lang="es">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Ejercicios</title>
    <style>
        body{
            text-align: center;
            font-family: 'Gill Sans MT';
            background-color: #7b7b7a;
            padding: 20px;
            color: #ffffff;
        }
        .boton{
            color: #7b7b7a;
            border-radius: 0.3em;
            padding: .5em 1em;
            border: none;
            background-color: #ffffff;
            cursor: pointer;
        }
    </style>
</head>
<body>
    <h2>Ejercicios</h2>
    <p>...</p>
    <form action="Ejercicio14.php" method="post">
        <input type="number" name="mes" placeholder="Introduce un mes">
        <input type="submit" class="boton" value="Enviar">
    </form>
    <?php
        if(isset($_POST['mes']) && !empty($_POST['mes'])) {
            $mes = intval($_POST['mes']); // Convertimos la entrada a un número entero
            
            switch($mes) {
                case 1:
                    echo '<p>Enero</p>';
                    break;
                case 2:
                    echo '<p>Febrero</p>';
                    break;
                case 3:
                    echo '<p>Marzo</p>';
                    break;
                case 4:
                    echo '<p>Abril</p>';
                    break;
                case 5:
                    echo '<p>Mayo</p>';
                    break;
                case 6:
                    echo '<p>Junio</p>';
                    break;
                case 7:
                    echo '<p>Julio</p>';
                    break;
                case 8:
                    echo '<p>Agosto</p>';
                    break;
                case 9:
                    echo '<p>Septiembre</p>';
                    break;
                case 10:
                    echo '<p>Octubre</p>';
                    break;
                case 11:
                    echo '<p>Noviembre</p>';
                    break;
                case 12:
                    echo '<p>Diciembre</p>';
                    break;
                default:
                    echo '<p>Mes no valido</p>';
                    break;
            }
        }
    ?>
    <p>FIN</p>
</body>
</html>

File: 11_Java/workspace DropBoxClase2017/TEMA2 Condicionales/src/Switch/Ejercicio14.java

package Switch;

import java.util.*;

/**
 * @author Ramon
 *
 */
public class Ejercicio14 {

	/**
	 * PDF-Ejercicio de Condicionales 14/22
	 * Descripcion:
	 * Condicionales Ejercicio con Switch.
	 * Switch
	 */
	public static void main(String[] args) {
		// 14.Haz un programa que pida un número de mes entre 1 y 12. A
		// continuación devolverá el nombre del mes correspondiente.

		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Introduzca un numero de mes entre 1 y 12: ");
		int opc = teclado.nextInt();
		
		
		switch(opc){
		case 1:
			System.out.println("Enero");
			break;
		case 2:
			System.out.println("Febrero");
			break;
		case 3:
			System.out.println("Marzo");
			break;
		case 4:
			System.out.println("Abril");
			break;
		case 5:
			System.out.println("Mayo");
			break;
		case 6:
			System.out.println("Junio");
			break;
		case 7:
			System.out.println("Julio");
			break;
		case 8:
			System.out.println("Agosto");
			break;
		case 9:
			System.out.println("Septiembre");
			break;
		case 10:
			System.out.println("Octubre");
			break;
		case 11:
			System.out.println("Noviembre");
			break;
		case 12:
			System.out.println("Diciembre");
			break;
		default:
			System.out.println("Error, Introduzca un numero de mes entre 1 y 12");
			break;
		}
		
		teclado.close();

	}

}
